po/ca.po

```diff
@@ -2790,7 +2790,7 @@ msgstr "Fallada transitòria"
 msgid "True"
 msgstr "Cert"
 
-#. TRANSLATORS: We verified the meatdata against the server
+#. TRANSLATORS: We verified the metadata against the server
```

Comment:
Some projects don't like changes to localization files... But it's about 50/50. (This was unintentional, I had excluded this file, so it shouldn't have been changed...)

Reply:
Maybe don't include .po please